tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+  * notice the "create-address" and "create-address" input fields
+
+
+* Coin.xml: At the top of the xml (after the version tag), include references to front-end files `app.css` and `mint.shtml`:
+  * (For reference, full TokenScript file provided -> `tutorial-files/Coin.xml`)
+```
+<!DOCTYPE token  [
+    <!ENTITY style SYSTEM "app.css">
+    <!ENTITY mint.en SYSTEM "mint.shtml">
+    ]>
+```
+
+  * Coin.xml: Inside the <ts:cards> section, add the following <ts:action>. The english name is "Mint", and front-end files referenced (style and view).
+```
+        <ts:action>
+            <ts:name>
+                <ts:string xml:lang="en">Mint</ts:string>
+            </ts:name>
+            ...
+            <!-- attributetype tags -->
+            ...
+            <!-- transaction tag -->
+            ...
+            <style type="text/css">&style;</style>
+            <ts:view>&mint.en;</ts:view>
+        </ts:action>
+```
+
+  * Coin.xml: To refer to the input types that will be used in a smart contract call, add their attribute-type (notice "create-address" and "create-amount")
+
+```
+			...
+            <ts:attribute-type id="create-address" syntax="1.3.6.1.4.1.1466.115.121.1.15">
+                <ts:name>
+                    <ts:string xml:lang="en">Address to create</ts:string>
+                </ts:name>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
+                    rationale for hardcoding: avoiding over-design -->
+                    <ts:user-entry as="address"/>
+                </ts:origins>
+            </ts:attribute-type>
+            <ts:attribute-type id="create-amount" syntax="1.3.6.1.4.1.1466.115.121.1.36">
+                <ts:name>
+                    <ts:string xml:lang="en">Amount to create</ts:string>
+                </ts:name>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
+                    rationale for hardcoding: avoiding over-design -->
+                    <ts:user-entry as="e18"/>
+                </ts:origins>
+            </ts:attribute-type>
+            ...
+```
+
+  * Coin.xml: Lastly add the transaction (refers to ethereum smart contract, function, and params in the data tag)
+```
+            <ts:transaction>
+                <ts:ethereum function="mint" contract="Coin">
+                    <ts:data>
+                        <ts:address ref="create-address"/>
+                        <ts:uint256 ref="create-amount"/>
+                    </ts:data>
+                </ts:ethereum>
+            </ts:transaction>
+```
+* Download the xml, css, shtml files, and jump to the section below to test: `Combining files and testing`
+  * The required `Makefile` for combining these files is in `tutorial-files`
+* Next steps: Once you have been able to test the Mint action, you'll notice the transaction is sent, but does not succeed because the wallet is not the minter.
```

Comment:
it succeeds but doesn't do anything. Why not include the mint function anyway?

Reply:
The mint function is in the Coin.sol example from Ethereum Studio. Try go through the tutorial, and see ;)